crates/type-system/README.md

```diff
@@ -0,0 +1,19 @@
+# The Block Protocol Type-System
+
+[//]: # (TODO: Introduction)
+
+## Requirements
+- [Rust](https://www.rust-lang.org/tools/install)
```

Comment:
Try running `yarn fix:markdownlint` and `yarn fix:prettier` from repo root to fix some formatting issues in this README.

Reply:
Was holding off until I figured out the appropriate ignores.

Applied the fix https://github.com/blockprotocol/blockprotocol/pull/503/commits/6dbd20e25655eb5eaa812509cb81a2ba38e36126

(and removed unnecessary README)